src/plugins/nonebot_plugin_cave_image_prompt/__main__.py

```diff
@@ -0,0 +1,159 @@
+from typing import cast
+
+from nonebot import on_message
+from nonebot.adapters import Event
+from nonebot.rule import Rule
+from nonebot_plugin_alconna import Image, Text, UniMessage
+from nonebot_plugin_larklang import LangHelper
+from nonebot_plugin_larkutils import get_user_id, is_private_message
+from nonebot_plugin_waiter import prompt_until
+
+lang = LangHelper()
+
+
+def is_single_image_message(event: Event) -> bool:
+    """检查消息是否只包含一张图片"""
+    try:
+        message = UniMessage.generate_without_reply(event=event)
+        
+        # 统计图片和文本数量
+        image_count = 0
+        text_content = ""
+        
+        for segment in message:
+            if isinstance(segment, Image):
+                image_count += 1
+            elif isinstance(segment, Text):
+                # 去除空白字符检查是否有实际文本内容
+                text_content += segment.text.strip()
+        
+        # 只有一张图片且没有文本内容
+        if image_count == 1 and not text_content:
+            return True
+    except Exception:
+        from nonebot.log import logger
+        logger.debug("Failed to check if message is single image", exc_info=True)
+    return False
+
+
+async def ask_cave_submission(user_id: str) -> bool:
+    """询问用户是否要投稿到 Cave"""
+    try:
+        response = await prompt_until(
+            UniMessage.text(await lang.text("cave_image_prompt.ask", user_id)),
+            [
+                await lang.text("cave_image_prompt.yes", user_id),
+                await lang.text("cave_image_prompt.no", user_id),
+            ],
+            timeout=60,
+        )
+        
+        yes_text = await lang.text("cave_image_prompt.yes", user_id)
+        return response and response.extract_plain_text().strip() == yes_text
+    except Exception:
+        return False
+
+
+# 监听私聊中的单图片消息
+image_prompt = on_message(
+    Rule(is_private_message) & Rule(is_single_image_message),
+    priority=10,
+    block=False,
```

Comment:
```suggestion
    priority=9,
    block=True,
```

先于 chat 插件响应并阻止事件传递到。chat 插件的 matcher